lib/initialEmotionEntries.js

```diff
@@ -1,51 +1,51 @@
-export const initialEmotionEntries = [
-  {
-    id: "1",
-    emotionType: "Anger",
-    intensity: "3",
-    notes: "Felt very frustrated during a meeting.",
-    dateTime: "2024-07-11T10:30:00",
-  },
-  {
-    id: "2",
-    emotionType: "Contempt",
-    intensity: "2",
-    notes: "Experienced contempt towards a colleague's unethical behaviour.",
-    dateTime: "2024-07-10T14:15:00",
-  },
-  {
-    id: "3",
-    emotionType: "Disgust",
-    intensity: "3",
-    notes: "Felt disgusted by the spoiled food in the fridge.",
-    dateTime: "2024-07-09T12:00:00",
-  },
-  {
-    id: "4",
-    emotionType: "Fear",
-    intensity: "3",
-    notes: "Worried about the upcoming job interview.",
-    dateTime: "2024-07-08T09:45:00",
-  },
-  {
-    id: "5",
-    emotionType: "Sadness",
-    intensity: "1",
-    notes: "Felt sad after watching a tragic movie.",
-    dateTime: "2024-07-07T20:00:00",
-  },
-  {
-    id: "6",
-    emotionType: "Surprise",
-    intensity: "3",
-    notes: "Surprised by an unexpected visit from an old friend.",
-    dateTime: "2024-07-06T16:30:00",
-  },
-  {
-    id: "7",
-    emotionType: "Enjoyment",
-    intensity: "3",
-    notes: "Had a great time at the beach with family.",
-    dateTime: "2024-07-05T13:00:00",
-  },
-];
+// export const initialEmotionEntries = [
+//   {
+//     id: "1",
```

Comment:
if you want to keep these for recreating emotions in your database, you could comment them in and remove the export